src/fs/statx.rs

```diff
@@ -10,6 +10,11 @@ use backend::fs::syscalls::statx as _statx;
 #[cfg(not(feature = "linux_4_11"))]
 use compat::statx as _statx;
 
+/// `STATX_ATTR_MOUNT_ROOT`—Address of the root of a mount
+#[allow(dead_code)]
```

Comment:
Since this is a `pub const` and re-exported in mod.rs, it shouldn't need `allow(dead_code)`.